### ISSUE TITLE
Item Data in Chat Component Hover Events do not display on 1.20.4 client on 1.20.6 server

### ISSUE THREAD

**Issue by XDMAN500** [ViaVersion/ViaBackwards#788]
### '/viaversion dump' Output

https://dump.viaversion.com/f607ade2ea7059165718592ab2768758d6125a478c523a89c8e5d34e753e0367

### Console Error

N/A

### Bug Description

The item data in chat hover events on 1.20.6 servers are not properly translated to 1.20.4 clients so item data like custom names, enchantments, attributes, and others are not visible to 1.20. clients.

### Steps to Reproduce

1. Start a server with 1.20.6
2. Join server with a 1.20.4 client
3. Execute command `tellraw @a {"text":"Hover me!","hoverEvent":{"action":"show_item","contents":{"id":"cake","count":1, "components":{"cu
stom_name":"{'text':'Special Cake Name',`color`:'yellow'}"}}}}`
4. The 1.20.4 client hovers over the chat message and only sees a default cake item
5. A 1.20.6 client client sees a cake with a yellow custom name "Special Cake name"

### Expected Behavior

The 1.20.4 client should see the "Special Cake Name" custom name on the cake item when hovering over the chat message.

### Additional Server Info

Appearance from a 1.20.4 client
![image](https://github.com/ViaVersion/ViaBackwards/assets/7269965/fee4ff0d-da5e-4964-a46a-af560d60ead2)

Appearance from a 1.20.6 client
![image](https://github.com/ViaVersion/ViaBackwards/assets/7269965/54636d01-9ef9-4e2c-af17-0cb19db0b6f8)

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. Velocity), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried a build from https://ci.viaversion.com/ and the issue still persists.

**Comment by Barvalg** [comment]
![Platform: 1.20.6--112--347bbe3%20%28MC%3A%201.20.6%29](https://img.shields.io/badge/Platform-1.20.6--112--347bbe3%20%28MC%3A%201.20.6%29-3498db?style=flat-square) </br>![ViaVersion (5.0.0): 4 commits behind master](https://img.shields.io/badge/ViaVersion%20(5.0.0)-4%20commit(s)%20behind%20master-yellow?style=flat-square) </br>![ViaBackwards(5.0.0): Even with master](https://img.shields.io/badge/ViaBackwards%20(5.0.0)-Even%20with%20master-brightgreen?style=flat-square) </br><br><h3>Please update **all** Via* plugins from [https://ci.viaversion.com/](https://ci.viaversion.com/).In case the issue still persists send the new dump and the issue will be reopened</h3>

**Comment by kennytv** [comment]
Known, but it'll take a bit to fix

**Comment by XDMAN500** [comment]
New dump added: https://dump.viaversion.com/f607ade2ea7059165718592ab2768758d6125a478c523a89c8e5d34e753e0367

**Comment by Barvalg** [comment]
![Platform: 1.20.6--112--347bbe3%20%28MC%3A%201.20.6%29](https://img.shields.io/badge/Platform-1.20.6--112--347bbe3%20%28MC%3A%201.20.6%29-3498db?style=flat-square) </br>![ViaVersion (5.0.1-SNAPSHOT): Even with master](https://img.shields.io/badge/ViaVersion%20(5.0.1--SNAPSHOT)-Even%20with%20master-brightgreen?style=flat-square) </br>![ViaBackwards(5.0.0): Even with master](https://img.shields.io/badge/ViaBackwards%20(5.0.0)-Even%20with%20master-brightgreen?style=flat-square) </br>

**Comment by Kryniowesegryderiusz** [comment]
Is there any eta for that? 👉👈

**Comment by FlorianMichael** [comment]
No, writing these conversions is quite time intensive and it will take a while, but I plan on doing it.

**Comment by qIooIp** [comment]
When can we expect this to be fixed? It's been a couple of months now.

**Comment by FlorianMichael** [comment]
Fixed in dev builds
